app/src/main/java/org/oppia/app/help/faq/FAQCategoryAdapter.kt

```diff
@@ -0,0 +1,78 @@
+package org.oppia.app.help.faq
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.R
+import org.oppia.app.databinding.FaqContentBinding
+
+/** The Recycler View adapter in the [FAQFragment]. */
+class FAQCategoryAdapter(
```

Comment:
This adapter implementation is incorrect. I think now we cannot skip the implementation of bindable adapter anymore considering that even the implementation of adapter is incorrect here. 

The best reference for your FAQ list is `StoryActivity` recyclerview implementation.

You can try this for some time but once you feel that you are not able to figure this out, feel free to inform me, so that I can work on this PR which you can check and learn from.

Reply:
> This adapter implementation is incorrect. I think now we cannot skip the implementation of bindable adapter anymore considering that even the implementation of adapter is incorrect here.
> 
> The best reference for your FAQ list is `StoryActivity` recyclerview implementation.
> 
> You can try this for some time but once you feel that you are not able to figure this out, feel free to inform me, so that I can work on this PR which you can check and learn from.

Okay I will start working on this.

Reply:
done :)